### PR TITLE
Use foreman-devel module

### PR DESCRIPTION
**Pull request by nofaralfasi**

### CODE REVIEW

**Comment by stejskalleos** [comment]
So, the error is clear:
```
RPM build errors:
error: Installed (but unpackaged) file(s) found:
   /usr/share/gems/gems/hammer_cli_foreman_ansible-0.5.0/locale/en/LC_MESSAGES/hammer_cli_foreman_ansible.mo
...
```
Checking the repository, `.mo` files are correctly added in the [spec file](https://github.com/theforeman/hammer-cli-foreman-ansible/blob/master/hammer_cli_foreman_ansible.gemspec#L16C1-L16C1), and we don't ignore them in the [.gitignore file](https://github.com/theforeman/hammer-cli-foreman-ansible/blob/master/.gitignore#L23). 

I don't understand why it's failing, @ofedoren @evgeni any thoughts?

**Comment by evgeni** [comment]
You probably need a similar change like we had in hammer_rh_cloud here to explicitly include the locale folder:
https://github.com/theforeman/foreman-packaging/pull/9864
https://github.com/theforeman/foreman-packaging/pull/9982

https://github.com/theforeman/foreman-packaging/blob/1b1d77395a48029a4b41f3e95f3cd726adfe1ff6/packages/plugins/rubygem-hammer_cli_foreman_ansible/rubygem-hammer_cli_foreman_ansible.spec#L54-L60

File: .packit.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
     trigger: pull_request
     targets:
       centos-stream-8:
-        additional_modules: "foreman:el8"
+        additional_modules: "foreman-devel:el8"
         additional_repos:
           - http://koji.katello.org/releases/yum/foreman-nightly/el8/x86_64/
```
Comment (evgeni) [inline .packit.yaml:34]: can you please update this to 
```suggestion
          - http://yum.theforeman.org/releases/nightly/el8/x86_64/
```

**Comment by evgeni** [comment]
technically, for hammer plugins, you do not need the `foreman-devel` module.

I think it's more correct to use it (and I'd merge this PR), but today the compilation should work with both just fine.

**Comment by evgeni** [comment]
And it was actually pointed out in https://github.com/theforeman/hammer-cli-foreman-ansible/pull/21:
> Packit fails the only fails I'm glad to observe :) indication that we finally will have translations 🤦 after we update rpmspec.

→ once you got a new gem released, we update the packaging and things will be green here too.

**Pull request merged**